scala-package/examples/src/main/scala/org/apache/mxnetexamples/Util.scala

```diff
@@ -42,4 +48,30 @@ object Util {
     }
    if (!success) throw new Exception(s"$url Download failed!")
   }
+
+  /**
+    * This Util is designed to manage the tests in CI
+    * @param name the name of the test
+    * @return runTest and number of epoch
+    */
+  def testManager(name: String) : (Boolean, Int) = {
```

Comment:
why is this needed ? can this not be controlled through make/maven targets and configurations?

Reply:
Then there will be 10+ lines maven configuration and fragments of checking in the tests to achieve the same functionalities. I believe placing them all together into one place is better to manage

Reply:
you'll need to change everytime a new test it added. Ideally your test should be configurable and you choose what settings you want to run in the target/script.

I don't think it should be controlled through the code here

Reply:
Let's say this way. If someday you want to add a new flag to split the test further for some other use cases, you need to open each test and add that flag there. Assume you have 10 tests in the plate, then 10 duplicated code change will be applied in the tests. This solution, one change would be necessary. Apart from that, this holds the same scenario of downloading files, we create a standard input and output into a single place will definite helps us for future maintenance.

Reply:
Agreeing with Naveen here. Convention is that we control this type of behavior through something like environment variables, build parameters, or config files.

One other thing is that we should try to avoid having code which is unique to our CI testing inside the src folder.

Reply:
isn't that why we have src/test/scala in the standard layout?

Reply:
Yes, you're right. I meant to say we should avoid having it in src/main folder.